app/src/main/AndroidManifest.xml

```diff
@@ -17,25 +17,26 @@
     <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
     <uses-permission android:name="com.google.android.apps.photos.permission.GOOGLE_PHOTOS" />
     <uses-permission android:name="android.permission.READ_LOGS" />
-    <uses-permission android:name="android.permission.SET_WALLPAPER"/>
+    <uses-permission android:name="android.permission.SET_WALLPAPER" />
 
 
     <!-- Needed only if your app targets Android 5.0 (API level 21) or higher. -->
     <uses-feature android:name="android.hardware.location.gps" />
 
     <application
         android:name=".CommonsApplication"
+        android:hardwareAccelerated="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/LightAppTheme"
         android:largeHeap="true"
-        android:hardwareAccelerated="false"
-        android:supportsRtl="true" >
-        <activity android:name="org.acra.CrashReportDialog"
-            android:theme="@android:style/Theme.Dialog"
-            android:launchMode="singleInstance"
+        android:supportsRtl="true"
+        android:theme="@style/LightAppTheme">
```

Comment:
If I understand correctly you just reordered things? Please restart your pull request from zero and only change things that are vital for the issue's resolution. Also, please split resolution of 2352 and resolution of 2245 into two separate pull requests, thanks! :-)